tests/acceptance/features/coreApiWebdavProperties/copyFile.feature

```diff
@@ -643,6 +645,27 @@ Feature: copy file
       | old              |
       | new              |
 
+
+  Scenario Outline: sharee copy a file into a shared folder, shared with viewer and secure viewer permissions
+    Given using <dav-path-version> DAV path
+    And user "Brian" has been created with default attributes and without skeleton files
```

Comment:
isn't it same as above scenario? `Scenario Outline: copying a file to a folder with no permissions`